doc/visualizations.rst

```diff
@@ -8,11 +8,19 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and input data (X, y).
+The `from_predictions` method creates a `Display` object when given the true and predicted values.
+This is useful when predictions should only be computed once.
+
+The `Display` object stores the computed values required for plotting with Matplotlib.
+These values can either be passed directly via `from_predictions`, or derived from an estimator
+and sample data using `from_estimator`.
+Additionally, the plot method allows adding to an existing plot via the ax parameter.
```

Comment:
```suggestion
Additionally, the plot method allows adding to an existing plot by passing the existing
plots :class`matplotlib.axes.Axes` to the `ax` parameter.
```

nitpick. I think (?) that API reference should work but best to check the rendered docs

Reply:
This :class`matplotlib.axes.Axes` doesn't render well @lucyleeow : 

<img width="719" alt="Screenshot 2025-02-14 at 10 37 24" src="https://github.com/user-attachments/assets/16201f48-12e3-4c2e-a582-22a6499340b9" />

Reply:
Ah yes typo,
```
:class:`matplotlib.axes.Axes`
``` 
I missed a :